HashRegistrarSimplified.sol

```diff
@@ -388,14 +388,21 @@ contract Registrar {
         entry h = _entries[_hash];
 
         h.value =  max(h.value, minPrice);
+        h.deed.setBalance(h.value, true);
 
-        // Assign the owner in ENS, if we're still the registrar
-        if(ens.owner(rootNode) == address(this))
-            ens.setSubnodeOwner(rootNode, _hash, h.deed.owner());
+        if (h.value >= 1000 ether) {
+            // Temporary registrar will not accept high value names yet
+            h.deed.closeDeed(995);
+            h.deed = Deed(0);
```

Comment:
This invalidation happens during an auction, whereas previously you could only invalidate a name after an auction had completed. Do we know how this will behave WRT other bids on the auction?